tests/unittests/test_main.py

```diff
@@ -204,16 +204,34 @@ def test_load_game_by_name(self):
 	pass
 
 
-@skip_todo
-def test_load_game_by_path(self):
-	pass
+@mock.patch('horizons.main.start_singleplayer')
+def test_load_game_by_path(mock_start_singleplayer):
+	"""
+	Test that a specific savegame file given by path can be loaded from the command line.
+	An in-game map is used instead of an actual savegame file - see NOTE above.
+	"""
+	start_game("--load-game", "content/save/savegame_test.sqlite")
+
+	options = mock_start_singleplayer.call_args[0][0]
+	assert not options.is_scenario
+	assert not options.is_map	# here the savegame is not treated as a loadable map
+	assert not options.is_editor
+	assert options.game_identifier == "content/save/savegame_test.sqlite"
 
 
 @skip_todo
 def test_edit_game_map_by_name(self):
 	pass
 
-
-@skip_todo
-def test_edit_game_map_by_path(self):
-	pass
\ No newline at end of file
+@mock.patch('horizons.main.start_singleplayer')
+def test_edit_game_map_by_path(mock_start_singleplayer):
+	"""
+	Test that a specific savegame file given by path be loaded into the editor from the command line.
+	An in-game map is used instead of an actual savegame file - see NOTE above.
+	"""
+	start_game("--edit-game-map", "content/save/savegame_test.sqlite")
```

Comment:
I actually didn't know about `tempfile`, but just read the docs and the module seems to be extremely useful.